Puppetfile

```diff
@@ -3,13 +3,10 @@ forge 'https://forgeapi.puppetlabs.com'
 mod 'katello/common',                '>= 1.0.1 < 1.1.0'
 mod 'katello/service_wait',          '>= 1.0.3 < 1.1.0'
 
-mod 'katello/certs',                 '>= 3.0.0 < 3.1.0'
-mod 'katello/qpid',                  '>= 3.1.0 < 3.2.0'
-mod 'katello/pulp',                  '>= 5.0.0 < 5.1.0'
-mod 'katello/candlepin',             '>= 4.0.0 < 4.1.0'
+mod 'katello/certs',                 '>= 3.0.1 < 3.0.2'
+mod 'katello/qpid',                  '>= 4.0.0 < 4.0.1'
+mod 'katello/pulp',                  '>= 5.3.0 < 5.4.0'
+mod 'katello/candlepin',             '>= 5.0.0 < 5.1.0'
 
-mod 'katello/foreman_proxy_content', '>= 6.0.0 < 6.1.0'
-mod 'katello/katello',               '>= 7.0.0 < 7.1.0'
-
-# Katello devel specific modules
-mod 'katello/katello_devel',         :git => 'https://github.com/Katello/puppet-katello_devel'
+mod 'katello/foreman_proxy_content', '>= 7.0.0 < 7.1.0'
+mod 'katello/katello',               '>= 7.2.0 < 7.3.0'
```

Comment:
All of the changes in this file are motivated by [this issue](http://projects.theforeman.org/issues/22289), which requires https://github.com/theforeman/puppet-foreman_proxy_content/commit/d1155d5f5405d23164ad0cb13580beb75bf98873 (7.0.0) and https://github.com/theforeman/puppet-qpid/commit/6a0450649716c317437c8e612fda2459fba8dd27 (4.0.0).